packages/bisect_ppx/bisect_ppx.0.2.2/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+name: "bisect_ppx"
+version: "0.2.2"
+maintainer: "Leonid Rozenberg <leonidr@gmail.com>"
+authors: "Leonid Rozenberg <leonidr@gmail.com>"
+homepage: "https://github.com/rleonid/bisect_ppx"
+dev-repo: "https://github.com/rleonid/bisect_ppx.git"
+bug-reports: "https://github.com/rleonid/bisect_ppx/issues"
+build: [
+  ["sh" "configure"]
+  [make "all"]
+  [make "install"]
```

Comment:
That's not possible. Either move the entire installation procedure to Makefile (use make "install" "BINDIR=%{bin}%" or something), or put everything in the .install file, which is just plantext with no scripting possible. I use the latter in my packages, with the .install file generated by topkg; you can take a look e.g. at https://github.com/whitequark/ppx_deriving_yojson (run make in the root of it) and use the resulting file as a template.